packages/web/src/pages/upload-page/UploadPageNew.tsx

```diff
@@ -99,7 +105,11 @@ export const UploadPageNew = () => {
             formState={formState}
             onContinue={(formState: UploadFormState) => {
               setFormState(formState)
-              setPhase(Phase.FINISH)
+              const hasPublicTracks =
+                formState.tracks?.some(
+                  (track) => !track.metadata.is_unlisted
+                ) ?? true
```

Comment:
if there are no tracks one could argue that there are no public tracks? 🤷
though I can see just keeping the default true to make sure we don't miss it

Reply:
I dont think you could even see this without having tracks in the form state, but I figured the public message would be a fine default 👍

Reply:
yea sounds good